src/components/Messages.js

```diff
@@ -0,0 +1,12 @@
+import React, {Component} from 'react'
+import ReactDOM from 'react-dom';
+
+class Messages extends React.Component{
+       render(){
+        return (
+        <h1>Messages page</h1>
```

Comment:
I realize that there's no dynamic data, yet, to show from the Firebase backend, but maybe this work would be more useful/helpful if some static "mock" data were to be displayed showing how messages will be visualized once they are available from the backend